### ISSUE TITLE
vscode doesn't see this module

### ISSUE THREAD

**Issue by Daxcor69** [steveukx/git-js#858]
Yes you can tell I am a noob.  I rely heavily on vscode to keep me out of trouble and make sure I use the 3rd party modules correctly.  I have tried to add this module to my common js code, and it would seem that vscode doesn't import the methods and properties so that I can have code completion.  Here is how I am instantiating the module as per docs

```
const simpleGit = require("simple-git");
simpleGit().clean(simpleGit.CleanOptions.FORCE);
simpleGit().addConfig("user.name", "Admin Pod").addConfig("user.email", "admin@server.com");
simpleGit("./mysource");
const git = simpleGit();
```
Then further in my code I use the following:
```
try {
      await git.init();
      await git.add(".");
      await git.commit(`Adding new version: ${req.body.version}`);
      await git.addRemote("main", "myremote url");
      await git.push("origin", "main");
    } catch (e) {
      console.error("error", e);
    }
```

The code above was just typed in and vscode never offered the different methods when I typed    `await git.`

What am I missing?

**Comment by steveukx** [comment]
Hello, it looks like you are almost there, the import is missing the destructuring though:


```
const { simpleGit } = require( 'simple-git' );
```

**Comment by Daxcor69** [comment]
Thank you that was issue. The one down side of following docs step by step is that you tend not to learn important facts about how to use a language when you only copy paste the usage examples. I saw the other examples however the common js example didn't show this destructuring. From the docs:

```
// require the library, main export is a function
const simpleGit = require('simple-git');
simpleGit().clean(simpleGit.CleanOptions.FORCE);

// or use named properties
const { simpleGit, CleanOptions } = require('simple-git');
simpleGit().clean(CleanOptions.FORCE);
```
it shows and either or.. so I went with the first one as that is what I am used to.  It doesn't show your suggestion only the second one does.  Not sure if the powers that be, maybe could edit the docs to include it.  thank you again for your help very much appreciated.
